### DISCUSSION TITLE
Hydrogeology: Provide Aquifer objects without geometry?

### DISCUSSION THREAD

**Discussion by JohannaOtt** [INSPIRE-MIF/helpdesk#142] [Q&A] [answered]
Our customer has data that could be mapped to the Aquifer type, but this data has no geometry information. If geometry information would be available, it would be delivered in MappedFeature object(s) but there is none.

Looking at the schema in the [UML model](https://inspire.ec.europa.eu/data-model/approved/r4618-ir/html/EARoot/EA2/EA2/EA2/EA1/EA7713.htm) and the [data specification](https://inspire.ec.europa.eu/file/1519/download?token=IGCGbum3), I would say that there is no formal requirement to provide MappedFeature object(s) with Aquifer object(s). So formally, this would be possible.

But from the domain's experts' point of view: Is there any point at all in delivering Aquifer data without geometry information or would you refrain from delivering that data in the INSPIRE context at all?

Any feedback is appreciated - thank you!

**Comment by fabiovinci** [comment, marked as answer]
Dear @JohannaOtt,

you are right, there are no specific constraints and/or requirements to provide the geometry for the Aquifer feature type.

However, looking at the description of the MappedFeature feature type it seems as if it should be mandatory:

![image](https://user-images.githubusercontent.com/22212406/203564512-ac5545eb-ca65-41d6-a7db-0a024e88c1d2.png)

It seems also confirmed by the Portrayal requirement since the provision of a compliant WMS requires the publication of the MappedFeature Spatial object type(s):
![image](https://user-images.githubusercontent.com/22212406/203564873-f71f31dd-3bce-40c0-ab71-e48b874c483b.png)
![image](https://user-images.githubusercontent.com/22212406/203578550-5add8205-7d3c-43ad-9981-c70dc31cec1a.png)

**Comment by JohannaOtt** [comment]
@fabiovinci Thank you for your assessment. I will recommend my customer not to transform their dataset to INSPIRE then.

Might make sense to add a sentence to the DS though so that all data providers follow the same approach.

**Comment by sMorrone** [comment]
Dear @JohannaOtt , I'm now converting this issue into a Q&A discussion for major visibility
